Bottom: 72e9836ee755017d3d5e2a4e2401c1d1560728bb
Top:    06eec99a7c26e00fbcda748f75172c4e833faca1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 15:02:07 +0100

hid/gtk: crosshair_{x,y} should be Coord, not int.


---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 6fe3a04..c3f00a3 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -176,8 +176,7 @@ typedef struct
   /* Viewport in PCB coordinates */
   Coord view_x0, view_y0, view_width, view_height;
   Coord pcb_x, pcb_y;
-
-  gint crosshair_x, crosshair_y;
+  Coord crosshair_x, crosshair_y;
 }
 GHidPort;
